src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -848,17 +848,15 @@ public void testImportGroupsRedundantSeparatedInternally() throws Exception {
         checkConfig.addAttribute("separated", "true");
         checkConfig.addAttribute("option", "bottom");
         final String[] expected = {
-            "5:1: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "org.*"),
+            "18:1: " + getCheckMessage(MSG_SEPARATED_IN_GROUP, "org.*"),
         };
-        verify(checkConfig, getNonCompilablePath("InputImportOrder_MultiplePatternMatches.java"),
+        verify(checkConfig, getNonCompilablePath("InputImportOrder_MultiplePatternMatches2.java"),
                 expected);
     }
 
     @Test
     public void testStaticGroupsAbove() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(ImportOrderCheck.class);
-        checkConfig.addAttribute("groups", "");
-        checkConfig.addAttribute("staticGroups", "");
```

Comment:
please do not change test configuration, please restore all lines like this

Reply:
I though these are redundant because `groups` and `staticGroups` are by default empty.

Reply:
done